src/components/list/examples/list-primary-component.tsx

```diff
@@ -0,0 +1,90 @@
+import { ListItem } from '@limetech/lime-elements';
+import { Component, h } from '@stencil/core';
+
+/**
+ * List with a primary component
+ */
+@Component({
+    tag: 'limel-example-list-primary-component',
+    shadow: true,
+})
+export class ListCircularProgressExample {
+    private actionItems: Array<ListItem<number>> = [
+        { text: 'Go to my fab object', value: 10 },
+        { text: 'Delete object', value: 11 },
+    ];
+
+    private items: Array<ListItem<number>> = [
+        {
+            text: 'King of Tokyo',
+            secondaryText: '2 players',
+            value: 1,
+            actions: this.actionItems,
+            primaryComponent: {
```

Comment:
I think this example is better without the extra complications of having a menu on one of the items.

```suggestion
    private items: Array<ListItem<number>> = [
        {
            text: 'King of Tokyo',
            secondaryText: '2 players',
            value: 1,
            primaryComponent: {
```

`fixup! docs(list): create a list example with a custom component`